test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpContextTests.cs

```diff
@@ -134,26 +134,7 @@ public void ItemsNotWrappedIfAlreadyImplementsIDictionary()
             Assert.Same(items, result);
         }
 
-        [Fact]
```

Comment:
let's keep this as we want it to come from the services. We can expose it through a static property, but internally we shouldn't rely on it that way